src/extension.ts

```diff
@@ -199,9 +198,10 @@ export function decorateLine(
  */
 // TODO: Use URLSearchParams instead of encodeURIComponent
 function buildUrl(errorQuery: string, sentryProjectId?: string): URL {
+    const sentryOrg = resolveSettings(sourcegraph.configuration.get<Settings>().value)['sentry.organization']
     const url = new URL(
         'https://sentry.io/organizations/' +
-            encodeURIComponent(SENTRYORGANIZATION!) +
+            encodeURIComponent(sentryOrg!) +
             '/issues/' +
             (sentryProjectId
                 ? '?project=' +
```

Comment:
Instead of concatenating strings for query params and using `encodeURIComponent()`, use `url.searchParams.set()`